scripts/symlink_core.py

```diff
@@ -15,4 +15,4 @@ def remove(path):
     remove(link_path)
     os.symlink(original_path,link_path)
 
-print("Linked local rbeezer/mathbook ")
+print("Linked local pretext directory")
```

Comment:
I'd slightly prefer "core pretext", what do you think?

Reply:
As this is just for developers, I agree, "Linked local core pretext directory" is clearer.